src/systems/filecoin_markets/storage_market/storage_market_actor.id

```diff
@@ -32,9 +37,9 @@ type StorageMarketActorState struct {
     // check if provider and client have sufficient balances
     _validateNewStorageDeal(rt Runtime, deal deal.StorageDeal) bool
 
-    _lockBalance(rt Runtime, addr addr.Address, amount actor.TokenAmount)
-    _unlockBalance(rt Runtime, addr addr.Address, amount actor.TokenAmount)
-    _transferBalance(
+    _rtLockBalance(rt Runtime, addr addr.Address, amount actor.TokenAmount)
```

Comment:
nit: as with the SPC's `lockPledgeCollateral`, having a more specific name, eg `lockStorageDealCollaterral` might be nice...

Reply:
lockBalance will also lock up storage fees here